project/SparkRedshiftBuild.scala

```diff
@@ -52,6 +52,7 @@ object SparkRedshiftBuild extends Build {
         "Spark 1.5.0 RC2 Staging" at "https://repository.apache.org/content/repositories/orgapachespark-1141",
       libraryDependencies ++= Seq(
         "com.amazonaws" % "aws-java-sdk-core" % "1.9.40" % "provided",
+        "com.amazonaws" % "aws-java-sdk-s3" % "1.9.40" % "provided",
```

Comment:
One risk: this adds a new dependency. This reminds me: where do the existing AWS SDK dependencies come from? Spark itself does not have a direct dependency on the AWS SDK and as far as I know neither does Hadoop.

Is this only marked as provided in order to avoid dependency conflicts in certain less-common scenarios? If so, maybe we should consider whether declaring this as a regular dependency would be be easier for more users. We just need to figure out whether dep. conflicts or missing deps are going to be more common here. /cc @jaley, any comments?